### PR TITLE
moved and tidied Manta, created tests

### PR DESCRIPTION
**Pull request by northwestwitch**
Tested on Rasta. Don't pay aso much attention to the untidied code on mip.pl (sub manta) because it's going to be a recipe, right?

### CODE REVIEW

File: lib/MIP/Program/Variantcalling/Manta.pm [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+}
+
+## Constants
+Readonly my $SPACE => q{ };
+
+sub manta_config {
+
+    ## manta_config
+
+    ## Function : Perl wrapper for writing Manta config recipe to $FILEHANDLE or return commands array. Based on Manta 1.0.0.
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Variantcalling/Manta.pm]: Based on Manta 1.0.0 -> Based on Manta 1.1.0

File: lib/MIP/Program/Variantcalling/Manta.pm [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+    );
+
+    return @commands;
+}
+
+sub manta_workflow {
+
+    ## manta_workflow
+
+    ## Function : Perl wrapper for writing Manta workflow recipe to $FILEHANDLE or return commands array. Based on Manta 1.0.0.
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Variantcalling/Manta.pm]: Same version change here

File: t/manta_config.t [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+use Modern::Perl qw{ 2014 };
+use warnings qw{ FATAL utf8 };
+use autodie;
+use 5.018;    #Require at least perl 5.18
+use utf8;
+use open qw{ :encoding(UTF-8) :std };
+use charnames qw{ :full :short };
+use Carp;
+use English qw{ -no_match_vars };
+use Params::Check qw{check allow last_error};
```
Comment (henrikstranneheim) [inline t/manta_config.t]: qw{check allow last_error}; -> qw{ check allow last_error };

File: t/manta_config.t [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+
+BEGIN {
+
+### Check all internal dependency modules and imports
+## Modules with import
+    my %perl_module;
+
+    $perl_module{q{Script::Utils}} = [qw{ help }];
+
+  PERL_MODULES:
```
Comment (henrikstranneheim) [inline t/manta_config.t]: Singular

File: t/manta_workflow.t [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+use Modern::Perl qw{ 2014 };
+use warnings qw{ FATAL utf8 };
+use autodie;
+use 5.018;    #Require at least perl 5.18
+use utf8;
+use open qw{ :encoding(UTF-8) :std };
+use charnames qw{ :full :short };
+use Carp;
+use English qw{ -no_match_vars };
+use Params::Check qw{check allow last_error};
```
Comment (henrikstranneheim) [inline t/manta_workflow.t]: same here. I will change the template

File: t/manta_workflow.t [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+
+BEGIN {
+
+### Check all internal dependency modules and imports
+## Modules with import
+    my %perl_module;
+
+    $perl_module{q{Script::Utils}} = [qw{ help }];
+
+  PERL_MODULES:
```
Comment (henrikstranneheim) [inline t/manta_workflow.t]: Singular

File: t/manta_workflow.t [hunk 7]
```diff
@@ -95,6 +95,9 @@ diag(   q{Test manta_workflow from Manta v}
       . $SPACE
       . $EXECUTABLE_NAME );
 
+## Base arguments
```
Comment (henrikstranneheim) [inline t/manta_workflow.t]: $function_base_command is duplicated

Comment (northwestwitch) [reply t/manta_workflow.t]: I did it that way on purpose , but I've modified it into something more similar to the other files now.

**Comment by henrikstranneheim** [comment]
👍

**Pull request merged**